package.json

```diff
@@ -46,13 +46,16 @@
     "eslint-plugin-react": "7.34.3",
     "eslint-plugin-react-hooks": "4.6.2",
     "gettext-parser": "8.0.0",
+    "glob": "7.2.3",
     "htmlparser": "1.7.7",
     "jed": "1.1.1",
+    "lodash": "4.17.21",
```

Comment:
This is only used for:
```
                const output = _.template(data.toString())({
                    title: test,
                    builddir: file.split("/").map(() => "../").join(""),
                    script: test + '.js',
                });
```

Time for a clever `replace()`.?

Reply:
```
[jelle@t14s][~/projects/cockpit/main]%du -hs node_modules/lodash
4.9M    node_modules/lodash
```

But ofcourse react-tables uses it.

Reply:
That's a nice idea though, then we at least lose the dependabot noise.

Reply:
Or alternative https://www.npmjs.com/package/lodash.template

Reply:
tbh lodash doesn't seem to get updates :)

Reply:
I like lodash and always think that I should use it more, but usually forget.  :)